README.md

```diff
@@ -379,6 +379,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Publish a package with 2FA using Slack](https://github.com/erezrokah/2fa-with-slack-action)
 - [Serialize Workflow Runs in Continuous Deployment Pipelines](https://github.com/softprops/turnstyle)
 - [Netlify Deploy GitHub Action for each commit](https://github.com/nwtgck/actions-netlify)
+- [Publish a Python distribution package to Anaconda Cloud](https://github.com/fcakyon/conda-publish-action)
```

Comment:
Please follow Title Casing rules for titles: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Reply:
> Please follow Title Casing rules for titles: https://github.com/sdras/awesome-actions/blob/master/contributing.md

should be fixed now